lib/worker/index.js

```diff
@@ -83,16 +88,53 @@ const sentryMiddleware = function (jobHandler) {
       throw err;
     }
   };
-};
+}
+
+/**
+ * Return an async function that sends timing data to NewRelic
+ *
+ * @param {Promise<object>} jobHandler - The normal job handler code function that we are wrapping
+ */
+function newrelicMiddleware(jobHandler) {
+  return async (job) => {
+    newrelic.startBackgroundTransaction(`job ${job.queue.name}`, 'worker queue', async () => {
```

Comment:
Is there any concern about this call failing for weird reasons?

Reply:
@srt32 "Shouldn't", but those are famous last words. This is wrapping the bull job, and this is using the standard API https://github.com/OptimalBits/bull/blob/develop/REFERENCE.md

That said, this is _also_ wrapped by our Sentry middleware. So if it does fail, it should be captured and raised to Sentry.

Great question.